poetry.lock

```diff
@@ -627,7 +627,7 @@ multidict = ">=5.0,<6.0"
 sanic-routing = ">=0.7,<1.0"
 ujson = {version = ">=1.35", markers = "sys_platform != \"win32\" and implementation_name == \"cpython\""}
 uvloop = {version = ">=0.5.3", markers = "sys_platform != \"win32\" and implementation_name == \"cpython\""}
-websockets = ">=10.0,<11.0"
```

Comment:
I think we need to keep this constraint on.

Reply:
Seems like this was a constraint never ported to 3.4.x 😞 it's present on 3.5.x and 3.6.x tho

Reply:
Yeah I vaguely remembered the fix on this branch was to directly amend `poetry.lock` although it's a brittle solution, since the lock file gets updated whenever we make changes to pyproject.

Reply:
I've added it manually and to pyproject.toml